src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -32,7 +33,7 @@ export interface ArtworkSidebarCommercialContainerProps
 export interface ArtworkSidebarCommercialContainerState {
   isCommittingCreateOrderMutation: boolean
   isErrorModalOpen: boolean
-  selectedEditionId: string
+  selectedEdition: any
```

Comment:
#minor if it could be typed with the actual properties of an edition set (which exists in the generated typings for the GraphQL fragment), that could be cool! I'm not 100% sure how to do that (refer to certain fields from the generated type here) cc @alloy

Reply:
Paired with @alloy on the types! :tada: